components-core/src/main/java/com/adyen/checkout/components/core/RedirectableActionComponent.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 1/8/2023.
+ */
+
+package com.adyen.checkout.components.core
+
+interface RedirectableActionComponent {
+
+    /**
+     * Set a callback that will be called when an redirect is opened.
```

Comment:
Typo: `a` redirect
Also maybe we can use "when a redirect is made" or something?